configure.ac

```diff
@@ -797,6 +797,9 @@ fi
 
 AM_CONDITIONAL([HAVE_LIBNL], [test "$have_libnl" = "yes"])
 
+AC_CHECK_LIB([rdmacm], [rdma_lib_reset],
+		[AC_DEFINE([RDMA_LIB_RESET], [1], [rdma_lib_reset enabled])], [])
```

Comment:
IMO, cannot use AC_CHECK_LIB(rdmacm) to check if it has symbol "rdma_lib_reset" - it will result in multiple "-lrdmacm" appear in the LDFLGAS and other duplicates. The right way to check if function is available with "AC_CHECK_FUNCS_ONCE (func)" and use HAVE_func in ifdef. 
It should be done after AC_CHECK_LIB(rdmacm) tested with some long-existing function to make sure rdmacm lib is loaded.